lib/parsers/entity_parser.js

```diff
@@ -340,6 +347,7 @@ function setSourceAssociationsForClass(relatedRelationships, entityName) {
 
       relationship.ownerSide = true;
     }
+    setOptionsOfRelationship(relatedRelationship, relationship);
```

Comment:
For here instead of "of", what do you think?

Reply:
Sure